.ci_scripts/environment.yml

```diff
@@ -19,3 +19,5 @@ dependencies:
   # uncomment to restore search
   # - elm
   # - nodejs
+  - pip: 
+    - sphinxcontrib-newsfeed # this helps when using vscode
```

Comment:
@isuruf is this okay or too impure now? We can comment it out and leave it there in case someone is using vscode --- it renders the html immediately and nicely, and somehow we don't have it in conda-forge.

<img width="1781" alt="Screen Shot 2022-01-22 at 15 17 48" src="https://user-images.githubusercontent.com/67342040/150654147-a5df56f2-3e86-4cf8-a4a1-36caa948f906.png">

Reply:
Remove this. We have a custom fork we use.

Reply:
Sure, just to be clear though, this is just for end users if they use vscode. However, I am highlighting it because it may mess up an automated system of creating the docs or something

Reply:
It will mess things up afaik.

Reply:
Yep, I will comment it out or delete it then, thanks!